setup.py

```diff
@@ -50,10 +50,13 @@ def get_packages(package):
     install_requires=["sqlalchemy>=1.4,<1.5", 'aiocontextvars;python_version<"3.7"'],
     extras_require={
         "postgresql": ["asyncpg"],
+        "asyncpg": ["asyncpg"],
+        "aiopg": ["aiopg"],
         "mysql": ["aiomysql"],
-        "mysql+asyncmy": ["asyncmy"],
+        "aiomysql": ["aiomysql"],
+        "asyncmy": ["asyncmy"],
         "sqlite": ["aiosqlite"],
-        "postgresql+aiopg": ["aiopg"],
```

Comment:
This will break pinned dependencies so I would change version of the lib to 0.6.0 in this commit.

That way, when the new release will be made you won't forget to bump the minor number. 

The package is published to pypi only with a new release anyway so it should be fine to bump the version upfront.

Reply:
I created a separate PR for this as we'll need to update changelog too.